modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your ServiceMeshControlPlane resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before you deploy your service mesh in production.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. For production deployments you _must_ customize the default template. 
+
+.Procedure
+
+. Configure Jaeger for production. 
+.. Edit the `ServiceMeshControlPlane` resource to use the `production` deployment strategy, by setting `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`. You can create and configure your Jaeger instance and set `spec.addons.jaeger.name` to the name of the Jaeger instance, for example `jaeger-production`.
```

Comment:
Elastisearch doesn't highlight like the rest of the names. Double check the ``

Also, I would put a comma before `jaeger-production`. So, `...to the name of Jaeger instance, for example, `jaeger-production.`